File: domain/src/test/kotlin/com/arturogutierrez/openticator/domain/category/model/CategoryTest.java
package com.arturogutierrez.openticator.domain.category.model;

import org.junit.Test;

import static org.hamcrest.CoreMatchers.is;
import static org.hamcrest.MatcherAssert.assertThat;

public class CategoryTest {

  @Test
  public void testCategoryModelFields() {
    Category category = new Category("id", "name");

    assertThat(category.getCategoryId(), is("id"));
    assertThat(category.getName(), is("name"));
  }
}

File: domain/src/test/kotlin/com/arturogutierrez/openticator/domain/category/model/CategoryTest.kt
package com.arturogutierrez.openticator.domain.category.model

import org.hamcrest.CoreMatchers.`is`
import org.hamcrest.MatcherAssert.assertThat
import org.junit.Test

class CategoryTest {

    @Test
    fun testCategoryModelFields() {
        val category = Category("id", "name")

        assertThat(category.categoryId, `is`("id"))
        assertThat(category.name, `is`("name"))
    }
}
